app/dashboard/models.py

```diff
@@ -2799,3 +2798,34 @@ class BlockedUser(SuperModel):
     def __str__(self):
         """Return the string representation of a Bounty."""
         return f'<BlockedUser: {self.handle}>'
+
+
+class FeedbackEntry(SuperModel):
+    bounty = models.ForeignKey(
+        'dashboard.Bounty',
+        related_name='feedbacks',
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True
+    )
+    sender_profile = models.ForeignKey(
+        'dashboard.Profile',
+        related_name='feedbacks_sent',
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True
+    )
+    receiver_profile = models.ForeignKey(
+        'dashboard.Profile',
+        related_name='feedbacks_got',
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True
+    )
+    rating = models.SmallIntegerField(blank=True, default=0)
+    comment = models.TextField(default='', blank=True)
+    feedbackType = models.TextField(default='', blank=True, max_length=20)
```

Comment:
It looks like there are only two types of feedback: `approver` and `worker`. If thats the case, what do you think about using the `choices` argument here to add a bit of validation and make the admin interface that much nicer? :)